utils/notification_service.py

```diff
@@ -1056,6 +1074,24 @@ def prepare_reports(title, header, reports, to_truncate=True):
     elif ci_event.startswith("Push CI (AMD)"):
         additional_files = {}
 
+    # A map associating the job names (specified by `inputs.job` in a workflow file) with the keys of
+    # `additional_files`. This is used to remove some entries in `additional_files` that are not concerned by a
+    # specific job. See below.
+    job_to_test_map = {
+        "run_pipelines_torch_gpu": "PyTorch pipelines",
+        "run_pipelines_tf_gpu": "TensorFlow pipelines",
+        "run_examples_gpu": "Examples directory",
+        "run_all_tests_torch_cuda_extensions_gpu": "Torch CUDA extension tests",
+        "run_tests_quantization_torch_gpu": "Quantization tests",
+    }
+
+    # Remove some entries in `additional_files` if they are not concerned.
+    test_name = None
+    job_name = os.getenv("CI_TEST_JOB")
+    if job_name in job_to_test_map:
+        test_name = job_to_test_map[job_name]
+    additional_files = {k: v for k, v in additional_files.items() if k == test_name}
```

Comment:
were we testing additional things in some jobs?

Reply:
No. That `additional` means the test jobs that are not the `modeling test jobs`: deepspeed, quantization, example, pipeline test jobs.